internal/graphql/graph/schema.resolvers.go

```diff
@@ -0,0 +1,118 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.27
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubeshop/testkube/internal/graphql/graph/model"
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+	executorsmapper "github.com/kubeshop/testkube/pkg/mapper/executors"
+	"github.com/kubeshop/testkube/pkg/rand"
+)
+
+// Type is the resolver for the type field.
+func (r *eventResolver) Type(ctx context.Context, obj *testkube.Event) (*string, error) {
+	panic(fmt.Errorf("not implemented: Type - type"))
+}
+
+// Resource is the resolver for the resource field.
+func (r *eventResolver) Resource(ctx context.Context, obj *testkube.Event) (*string, error) {
+	panic(fmt.Errorf("not implemented: Resource - resource"))
+}
+
+// Features is the resolver for the features field.
+func (r *executorResolver) Features(ctx context.Context, obj *testkube.Executor) ([]*model.FeaturesListItem, error) {
+	panic(fmt.Errorf("not implemented: Features - features"))
+}
+
+// Labels is the resolver for the labels field.
+func (r *executorResolver) Labels(ctx context.Context, obj *testkube.Executor) (*string, error) {
+	panic(fmt.Errorf("not implemented: Labels - labels"))
+}
+
+// Event is the resolver for the event field.
+func (r *executorDetailsResolver) Event(ctx context.Context, obj *testkube.ExecutorDetails) (*testkube.Event, error) {
+	panic(fmt.Errorf("not implemented: Event - event"))
+}
+
+// Tooltips is the resolver for the tooltips field.
+func (r *executorMetaResolver) Tooltips(ctx context.Context, obj *testkube.ExecutorMeta) (*string, error) {
+	panic(fmt.Errorf("not implemented: Tooltips - tooltips"))
+}
+
+// Executors is the resolver for the executors field.
+func (r *queryResolver) Executors(ctx context.Context) ([]*testkube.Executor, error) {
+	panic(fmt.Errorf("not implemented: Executors - executors"))
+}
+
+// Executors is the resolver for the executors field.
+func (r *subscriptionResolver) Executors(ctx context.Context) (<-chan []testkube.ExecutorDetails, error) {
+	ch := make(chan []testkube.ExecutorDetails)
+
+	// TODO You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
+	// For this example we'll simply use a Goroutine with a simple loop.
+	go func() {
+		r.Log.Infof("%+v\n", "subscribed to events.executor.>")
+
+		r.Bus.SubscribeTopic("events.executor.>", rand.String(30), func(e testkube.Event) error {
+			r.Log.Infof("%s %s %s\n", e.Type_, *e.Resource, e.ResourceId)
+
+			execs, err := r.Client.List("")
```

Comment:
will we send there always all items regardless of filtering?

Reply:
probably not? don't know yet, we should define what params are needed with @rangoo94 and @pavloburchak for now whole list (for executors there is no filtering??)
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/30776/228303664-cf985924-b36d-47c1-8220-52ed43927768.png">

Reply:
We do not have any filtering yet, and I don't think that someone will have so many custom executors that it will be a problem. I would say that it may be easily postponed unless you would like to experiment with GraphQL variables here.